_delphi_utils_python/delphi_utils/runner.py

```diff
@@ -41,7 +41,9 @@ def run_indicator_pipeline(indicator_fn:  Callable[[Params], None],
     archiver = archiver_fn(params)
     if validator:
         validation_report = validator.validate()
-        validation_report.log(get_structured_logger(params["common"].get("log_filename", None)))
+        validation_report.log(get_structured_logger(
+            __name__,
```

Comment:
What do the validation logs that did run have for the logger name? What's your hypothesis for why the wrong logger name would make indicators run via the runner not have logs saved?

Reply:
get_structured_logger() takes name as the first argument, so filename is none and therefore the validation report isn't getting saved.

Reply:
I think keeping it at __name__ or indicator_fn.__module__ would be similar since I've been retrieving individual validator logs from the filename rather than the logger name - could still definitely make the change, though! Thoughts?

Reply:
> get_structured_logger() takes name as the first argument, so filename is none and therefore the validation report isn't getting saved.

Hm, I'm not following. Where does the filename come from? If it's from the `name` arg to `get_structured_logger()`, it shouldn't ever be `None`.

`get_structured_logger()` has `name = __name__` as default. This corresponds to `delphi_utils` since Python arg defaults are evaluated when a function is defined.

Reply:
The original call only gave one argument and didn't specify which keyword it was, so `params["common"].get("log_filename", None)` became the name and filename was none

Reply:
👌  Wow, subtle. Thanks for the explanation.

> keeping it at `__name__` or `indicator_fn.__module__` would be similar since I've been retrieving individual validator logs from the filename rather than the logger name

Got it. So if the filename already indicates the indicator, do we also want the logger name to be the indicator module or to be the more generic `delphi_utils.runner`? I'm not sure what the logger name is used for. @krivard

Reply:
@qx-teo Go with `indicator_fn.__module__`, please.

Reply:
the logger name is used in the kibana dashboards, so it's easiest if the validator logs and the indicator function logs match.

Reply:
Gotcha! Just tested locally  too, using `indicator_fn.__module__` should make it match